generator/ServiceClientGeneratorLib/Generators/SourceFiles/ServiceClientsNetFramework.tt

```diff
@@ -375,6 +375,12 @@ namespace <#=this.Config.Namespace#>
 
         foreach(var operation in this.Config.ServiceModel.Operations)
         {
+            if(operation.IsInternal || 
+				((this.Config.ServiceModel.H2Support == H2SupportDegree.Optional && operation.IsEventStreamBidi) ||
+				(this.Config.ServiceModel.H2Support == H2SupportDegree.EventStream && operation.IsEventStreamOutput))
```

Comment:
We are missing the third state of `H2Support` being set to `Required. The third state is probably missing in the .NET Framework interface T4 as well.

Reply:
Fixed all of these.